plugins/inputs/gtfs/README.md

```diff
@@ -0,0 +1,81 @@
+# GTFS
+
+This input plugin can be used to gather vehicle positions, trip updates and
+service alerts from a GTFS-realtime feed. [GTFS-realtime](https://developers.google.com/transit/gtfs-realtime) 
+is an extension of the [General Transit Feed Specification](https://developers.google.com/transit/gtfs).
+
+### Configuration
+
+This plugin can be configured to collect any combination of vehicle positions, trip updates
+and service alerts. Enable any one of these three input types by populating its URL in the
+plugin configuration. At least one of the three must be configured.
+
+```toml
+[[inputs.gtfs]]
+  ## API Key
+  # key = "${GTFS_API_KEY}"
+  ## URL for fetching vehicle positions
+  # vehicle_positions_url = "https://host.test/VehiclePositions.pb"
+  ## URL for fetching vehicle positions
+  # trip_updates_url = "https://host.test/TripUpdates.pb"
+  ## URL for fetching vehicle positions
+  # service_alerts_url = "https://host.test/ServiceAlerts.pb"
```

Comment:
You maybe should add a base-url here or is it a common use-case to use different servers to serve those?
Furthermore, can the suffices (`VehiclePositions.pb`, `TripUpdates.pb` and `ServiceAlerts.pb`) actually vary? If not it would be better to move them to the code...

Reply:
I am mostly developing this plugin against Boston transit system data. I am not sure how other systems expose these endpoints. I will look into this, but unless I find is a clear standard, I would like to provide ultimate flexibility to accommodate any transit system using GTFS-realtime, even if they use different suffixes.

Reply:
Well the issue is that you need to (potentially) provide three TLS configs in this case. I guess if you have different servers it would be easier to duplicate the plugin instance in the telegraf config...